serverless/sql-databases/reference-content/known-differences.mdx

```diff
@@ -84,7 +84,7 @@ If you require strict PostgreSQL features compatibility, you can use [Managed da
     RESET timezone;
```

Comment:
In the part related to DDL queries recall what DDL is (it's an advanced database concept / wording).
"- DDL queries on objects (...)" -> "- DDL (Data Definition Language) queries on objects (...)"